.eslintrc

```diff
@@ -20,7 +20,7 @@
     },
     "extends": "eslint:recommended",
     "parserOptions": {
-        "ecmaVersion": 6,
+        "ecmaVersion": 2017,
```

Comment:
This PR introduces a test using `async`/`await` which necessitated changing the parserOptions for ESLint.